### ISSUE TITLE
Panel with DetailsList in Scrollable pane does not render DetailsList after initial render.

### ISSUE THREAD

**Issue by sumboxcar182** [OfficeDev/office-ui-fabric-react#5336]
### Bug Report 
 
- __Package version(s)__: 5.111.0 (Also 6.19.0)
- __Browser and OS versions__: I don't believe it's relevant, but i'm running Chrome(67.0.3396.87) and Windows 10 
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: (Blocking, High, Normal, Low): High
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
 
We are currently hitting an issue when trying to render a ScrollablePane with a DetailsList on a Panel. 

When we try to do this, it loads perfectly the first time, but every time we bring up the Panel again, we are seeing a blank area where the DetailsList should be. Inspecting the DOM it's showing that the ScrollablePane is there, but the only piece of the DetailsList that is there is the Viewport div.  If we resize the window, the DetailsList appears correctly. 

#### Actual behavior:
The first time we open up a Panel, the table is rendered correctly.

Every time after our initial render, the  Panel no longer renders the table, although the rest of the components on the Panel are rendered ok. 

 
#### Expected behavior:
 When we bring up the panel, at any point after the initial render, it should still display the DetailsList.

 
### If applicable, please provide a codepen repro:
 My hope is to have one by end of the week, we're in crunch mode at work and i don't have time as of this moment to get one..

**Comment by dzearing** [comment]
One workaround; when you collapse the panel, it's not removing DOM, is my guess. Can you remove the Panel (e.g. conditionalize it) when its closed?

Alternatively:

1. force a resize event on window on expansion
2. forceUpdate on the detailslist

Ultimately because we don't get any sort of re-render or resize event in this case, the details list isn't reevaluating the viewport. So we need to consider some of these workarounds.

**Comment by sumboxcar182** [comment]
Hi and my apologies in the delay for replying but thank you so much for the advice!

 We found the issue was that the container div that wraps the component did not have an exact height, so if the DetailsList had a wrapper with height as 100% for example, it was breaking. So taking your advice we ended up just having a resize handler that would check for any resize event and would set the style to be equal to an absolute height, which ended up fixing this issue. 

I think this PR is safe to close as i do not believe it's an issue on your end. 

Again thank you so much for the help!
